packages/firebase_data_connect/firebase_data_connect/example/lib/main.dart

```diff
@@ -109,8 +109,14 @@ class _DataConnectWidgetState extends State<DataConnectWidget> {
   void initState() {
     super.initState();
 
-    QueryRef<ListMoviesResponse, void> ref =
+    QueryRef<ListMoviesData, void> ref =
         MoviesConnector.instance.listMovies.ref();
+    // MoviesConnector.instance.addDateAndTimestamp
```

Comment:
Still need to keep those commented out code?

Reply:
Removed